datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -1072,6 +1125,64 @@ input DatasetUpdateInput {
     Update to institutional memory, ie documentation
     """
     institutionalMemory: InstitutionalMemoryUpdate
+
+    """
+    Update to global tags
+    """
+    globalTags: GlobalTagsUpdate
+
+    """
+    Update to editable schema metadata of the dataset
+    """
+    editableSchemaMetadata: EditableSchemaMetadataUpdate
+}
+
+input EditableSchemaMetadataUpdate {
+    editableSchemaFieldInfo: [EditableSchemaFieldInfoUpdate!]!
+}
+
+input EditableSchemaFieldInfoUpdate {
+    """
+    Flattened name of a field identifying the field the editable info is applied to
+    """
+    fieldPath: String
+
+    """
+    Edited description of the field
+    """
+    description: String
+
+    """
+    Tags associated with the field
+    """
+    globalTags: GlobalTagsUpdate
+}
+
+input GlobalTagsUpdate {
+    tags: [TagAssociationUpdate!]
+}
+
+input TagAssociationUpdate {
+    tag: TagUpdate!
+}
+
+input TagUpdate {
+    urn: String!
```

Comment:
Looks like we do have the urn! We can use this above instead of the "name" field